r2/r2/models/link.py

```diff
@@ -217,6 +217,12 @@ def _submit(cls, is_self, title, content, author, sr, ip,
             url = content
             selftext = cls._defaults["selftext"]
 
+        nsfwcheck = re.compile(r"\bnsf[wl]\b", re.I)
+        over_18 = False
+
+        if bool(nsfwcheck.search(title)):
```

Comment:
Really minor thing, but since you're going to have to change this line anyway, you can drop the `bool()` conversion. Just `if cls._nsfw.search(title):` is good.